patches/tModLoader/Terraria/Program.TML.cs

```diff
@@ -86,18 +84,30 @@ public static partial class Program
 						File.Copy(Path.Combine(releasePath, "input profiles.json"), Path.Combine(newPath, "input profiles.json"));
 				}
 			}
-			*/
+		}
+
+		private static void Port143FilesFromStable(string savePath, bool isCloud) {
+			// Copy all current stable player files to 1.4.3-legacy during transition period. Skip ModSources & Workshop shared folders
 
-			// Copy all current stable player files to 1.4.3-legacy during transition period
-			if (!Directory.Exists(Path.Combine(SavePath, ReleaseFolder)) && Directory.Exists(Path.Combine(SavePath, StableFolder))) {
-				Utilities.FileUtilities.CopyFolder(Path.Combine(SavePath, StableFolder), Path.Combine(SavePath, ReleaseFolder));
+			string newFolderPath = Path.Combine(savePath, ReleaseFolder);
+			string oldFolderPath = Path.Combine(savePath, StableFolder);
+
+			if (!Directory.Exists(newFolderPath) && Directory.Exists(oldFolderPath)) {
+				Utilities.FileUtilities.CopyFolder(oldFolderPath, newFolderPath, isCloud,
+					excludeFilter: new System.Text.RegularExpressions.Regex("(Workshop)|(ModSources)"));
 			}
 		}
 
+		internal static void PortFilesMaster(string savePath, bool isCloud) {
+			PortOldSaveDirectories(savePath);
+			PortCommonFilesToStagingBranches(savePath);
+			Port143FilesFromStable(savePath, isCloud);
+		}
 
 		public const string ReleaseFolder = "tModLoader-1.4.3";
```

Comment:
Is `ReleaseFolder` a misnomer here? `LegacyFolder` better?

Reply:
Selected Legacy143Folder for best clarity